app/controllers/admins/employees_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Admins
   class EmployeesController < BaseController
     def index
-      @employees = Employee.all.order('id ASC')
```

Comment:
In my honest opinion, this change makes the code less readable.
At first, I started wondering how you are passing this variable to view then I understood that this is the method. When I scrolled down, I saw that this method returns an instance variable and that is the way you are passing this to view. 
But at first glance, it wasn't obvious what is happening in this index method.